Subject: [PATCH] Fix tests failing due to extra message from built-in web server
Remove messages like:

    PHP 7.1.4 Development Server started at Mon May  1 00:42:39 2017

from test outputs, because tests do not expect them.

--- a/run-tests.php 2020-10-23 17:52:11.921218721 +0100
+++ b/run-tests.php 2020-10-23 17:53:30.521489872 +0100
@@ -2500,6 +2500,9 @@
     // Does the output match what is expected?
     $output = preg_replace("/\r\n/", "\n", trim($out));
 
+    // Remove message from built-in development server.
+    $output = preg_replace("/^PHP [0-9.]+ Development Server started at .*\n\n?/m", "", $output);
+
     /* when using CGI, strip the headers from the output */
     $headers = array();
 
